lib/team.rb

```diff
@@ -57,40 +57,20 @@ def total
     }
   end
 
-  def win_percentage
-    (total[:wins] / total[:games].to_f).round(2)
-  end
-
-  def home_win_percentage
-    (home[:wins] / home[:games].to_f).round(2)
-  end
-
-  def away_win_percentage
-    (away[:wins] / away[:games].to_f).round(2)
+  def win_percentage(type)
+    (type[:wins] / type[:games].to_f).round(2)
   end
```

Comment:
See now that the history[:home]/history[:away] will need to go in league_stats when the method is called. Not here.